docs/book/src/getting-started/testdata/project/cmd/main.go

```diff
@@ -98,9 +96,10 @@ func main() {
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 		Scheme: scheme,
 		Metrics: metricsserver.Options{
-			BindAddress:   metricsAddr,
-			SecureServing: secureMetrics,
-			TLSOpts:       tlsOpts,
+			BindAddress:    metricsAddr,
+			FilterProvider: filters.WithAuthenticationAndAuthorization,
```

Comment:
Could we add a comment explaining what it does?
Add the options to pass the certs commented and says that for production it is recommended use real certs instead  and/or use cert-manager?